src/azure/ApiCenter/ApiCenterService.ts

```diff
@@ -159,16 +159,22 @@ export class ApiCenterService {
     };
     let response = await client.sendRequest(options);
 
-    const location = response.headers.get("Location");
+    if (response.status === 200) {
+      return true;
+    } else if (response.status === 202) {
+      const location = response.headers.get("Location");
 
-    if (!location) { return false; }
+      if (!location) { return false; }
 
-    options = {
-      method: "GET",
-      url: location,
-    };
+      options = {
+        method: "GET",
+        url: location,
+      };
 
-    return await this.sendRequestWithTimeout(client, options, 120000); // 2 minutes in milliseconds
+      return await this.sendRequestWithTimeout(client, options, 120000); // 2 minutes in milliseconds
+    } else {
+      return false;
```

Comment:
shall we give information to users about the error?

Reply:
If it is 4xx or 5xx, `client.sendRequest(options)` should throw error in line 160. So for all the known cases, we would not go to this line of code, unless there is a unknown case.